cjs/index.js

```diff
@@ -3,6 +3,7 @@
 // Runtimes still being tested out can be added, just not at runtimes[runtime][0]
 let runtimes = {
   node: [
+    'nodejs16.x',
```

Comment:
Should this be in the second position of the array so we can test out everything before making it the default?

Reply:
That's a really good question. We can discuss in our voice meeting and update here as needed.